book/src/user/docker.md

```diff
@@ -59,19 +59,22 @@ All available Cargo features are listed at
 
 ## Configuring Zebra
 
-To configure Zebra, edit the `docker/default-zebra-config.toml` config file and
-uncomment the `configs` mapping in `docker/docker-compose.yml` so your config
-takes effect. You can see if your config works as intended by looking at Zebra's
-logs.
-
-Alternatively, you can configure Zebra by setting the environment variables in
-the `docker/.env` file. Note that the config options of this file are limited to
-the variables already present in the commented out blocks in it and adding new
-ones will not be effective. Also note that the values of the variables take
-precedence over the values set in the `docker/default-zebra-config.toml` config
-file. The `docker/.env` file serves as a quick way to override the most commonly
-used settings for Zebra, whereas the `docker/default-zebra-config.toml` file
-provides full config capabilities.
+To configure Zebra, you have three options:
+
+1. Edit the `docker/default-zebra-config.toml` config file and uncomment the `configs` mapping in `docker/docker-compose.yml` so your config takes effect.
```

Comment:
This documentation is outdated.

Reply:
Fixed here: [chore: align documentation and implementation](https://github.com/ZcashFoundation/zebra/pull/9344/commits/2520086bd2effaefe7ee2ddf7caa684d3d897904)